tests/conftest.py

```diff
@@ -1 +1,9 @@
 """Test configuration for pytest."""
+import asyncio
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def event_loop():
+    return asyncio.new_event_loop()
```

Comment:
why is this needed?

Reply:
We need the event loop to handle the async funcs (like in Contracts's tests)